pkg/controller/k8srole.go

```diff
@@ -9,21 +9,22 @@ import (
 
 type k8srole struct {
 	*DependentResourceHelper
+	reconciler *BaseGenericReconciler
```

Comment:
Why are you injecting the reconciler when it doesn't appear to be used anywhere in the code?

Reply:
I simply adapted the code according to last code committed on issue-80 but didn't has the chance to check if it was used, needed, ...

Reply:
Will remove it then